src/main/java/net/minecraftforge/fml/event/server/FMLServerStartingEvent.java

```diff
@@ -19,14 +19,13 @@
 
 package net.minecraftforge.fml.event.server;
 
-import com.mojang.brigadier.CommandDispatcher;
-import net.minecraft.command.CommandSource;
 import net.minecraft.server.MinecraftServer;
 
 /**
  * Called after {@link FMLServerAboutToStartEvent} and before {@link FMLServerStartedEvent}.
- * This event allows for customizations of the server, such as loading custom commands, perhaps customizing recipes or
- * other activities.
+ * This event allows for customizations of the server.
+ *
+ * If you need to add commands or change recipes, see {@link net.minecraftforge.event.RegisterCommandsEvent} and {@link net.minecraftforge.event.AddReloadListenerEvent} respectively.
```

Comment:
Telling people to change recipes in code is a non-starter.

Reply:
Only mentioned recipes since that is what was initially there. 

Updated it though.